source/js/GliderRecipe.js

```diff
@@ -0,0 +1,201 @@
+class GliderRecipe extends HTMLElement {
+  constructor(){
+    super();
+    let shadow = this.attachShadow({mode: 'open'});
+  }
+
+  set data(data) {
+
+    let cleanData = {};
+    /** 
+     * Making a more organized object for getting 
+     * the info from messy recipe json. Might not end up using all of it,
+     * I just yoinked this from my lab code
+     * 
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8437408)
```suggestion
     *
```